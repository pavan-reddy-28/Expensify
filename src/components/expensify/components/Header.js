import React from 'react'
import {NavLink} from 'react-router-dom'
const Header = () => (
    <div >
        <h1>
            Expensify
</h1>
        <NavLink className="nav-link-el" to="/" activeClassName="is-active" exact> DashBoard</NavLink>
       
        <NavLink className="nav-link-el" to="/create" activeClassName="is-active">Create</NavLink>
        <NavLink className="nav-link-el" to="/help" activeClassName="is-active">Help</NavLink>

    </div>
)


export default Header
