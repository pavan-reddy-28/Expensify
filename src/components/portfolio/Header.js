import React from 'react'
import {NavLink} from 'react-router-dom'
const Header = () => (
    <div >
        <h1>
           PortFolio
</h1>
        <NavLink className="nav-link-el" to="/" activeClassName="is-active" exact> Home</NavLink>
        <NavLink className="nav-link-el" to="/portfolio" activeClassName="is-active">Portfolio</NavLink>
        <NavLink className="nav-link-el" to="/help" activeClassName="is-active">Contact</NavLink>
   

    </div>
)


export default Header
