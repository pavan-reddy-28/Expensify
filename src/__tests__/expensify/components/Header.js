import React from 'react'
// import ReactShallowRenderer from 'react-test-renderer/shallow'; // ES6
import Header from '../../../components/expensify/components/Header'
import {shallow} from 'enzyme'
test('should RENDER HEADER CORRECTLY', () => {

    const wrapper  = shallow(<Header/>);
    expect(wrapper.find('h1').text()).toBe("Expensify");
    expect(wrapper).toMatchSnapshot();
})

