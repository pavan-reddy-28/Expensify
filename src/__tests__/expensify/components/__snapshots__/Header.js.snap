// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should RENDER HEADER CORRECTLY 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    className="nav-link-el"
    exact={true}
    to="/"
  >
     DashBoard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    className="nav-link-el"
    to="/create"
  >
    Create
  </NavLink>
  <NavLink
    activeClassName="is-active"
    className="nav-link-el"
    to="/help"
  >
    Help
  </NavLink>
</div>
`;
